core/templates/components/site-search/site-search.twig

```diff
@@ -24,7 +23,7 @@
  <form action="{{ action }}" method="{{ method }}" accept-charset="UTF-8">
   <label class="su-site-search__sr-label" for="{{ search_input_name|default('search-field') }}">{{ search_label|default('Search this site') }}</label>
   <input {{- input_attributes }} type="text" id="{{ search_input_name|default('search-field') }}" name="{{ search_input_name|default('search-field') }}" class="su-site-search__input" placeholder="{{- placeholder -}}" maxlength="128">
-  <button {{- button_attributes }} type="submit" name="submit" class="su-site-search__submit" aria-label="Search"><img src="{{ search_icon_src }}" alt="Search button" /></button>
+  <button {{- button_attributes }} type="submit" name="submit" class="su-site-search__submit su-sr-only-text" aria-label="Search">Submit Search</button>
```

Comment:
Interesting application of `su-sr-only-text`: It seems to move the text within the `<button>` but not the button itself. That's super cool if it works cross-browser. Does it work in all supported browsers?

Reply:
Yup I've checked as far back as IE 10.

Reply:
Awesome! Neat trick!